core/src/main/java/com/yahoo/imapnio/async/data/ExtensionMailboxInfo.java

```diff
@@ -15,9 +15,18 @@
     /** Literal for MAILBOXID. */
     private static final String MAILBOX_ID = "MAILBOXID";
 
+    /** Literal for CLOSED. */
+    private static final String CLOSED_ID = "CLOSED";
+
     /** Variable to store mailbox Id. */
     private String mailboxId;
 
+    /** Variable to store previously selected mailbox was closed implicitly. */
+    private boolean closed;
+
+    /** Tagged response when the status is OK. */
+    private IMAPResponse taggedResponse;
```

Comment:
taggedOkResponse

Reply:
When do you need this response?

Reply:
This is the response that will have when "Sorry UIDMismatch".

Reply:
The base class MailboxInfo has a responses (List<IMAPResponse>) member either you can add to that or expose a boolean. This looks a bit too specific for a usecase.

Reply:
I can add to existing list.